src/main/java/org/folio/circulation/item/domain/mapper/CirculationItemsMapper.java

```diff
@@ -13,18 +13,17 @@
 @Mapper(componentModel = "spring", nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS)
 public interface CirculationItemsMapper {
 
-  @Mappings({ @Mapping(target = "id", source = "id"),
-      @Mapping(target = "holdingsRecordId", source = "holdingsRecordId"),
-      @Mapping(target = "status", source = "status"),
-      @Mapping(target = "materialTypeId", source = "materialTypeId"),
-      @Mapping(target = "permanentLoanTypeId", source = "permanentLoanTypeId"),
-      @Mapping(target = "instanceTitle", source = "instanceTitle"),
-      @Mapping(target = "itemBarcode", source = "itemBarcode"),
-      @Mapping(target = "pickupLocation", source = "pickupLocation")})
+      @Mapping(target = "id", source = "id")
```

Comment:
Since the source and target are same, I think it will work even without this @Mapping annotation. Please check